index.html

```diff
@@ -144,10 +106,6 @@ <h3 class="newsletter">Subscribe to Our Newsletter</h3>
 		<p>&copy; 2015 optiMize</p>
 	    </footer>
 	</div>
-	<script>
-	 $(window).load(function(){
-	     $('.load').hide();
-	 });
-	</script>
```

Comment:
@john-dickinson can you move the script tag into the body?
exactly at this location. thx